### DISCUSSION TITLE
Key to images used

### DISCUSSION THREAD

**Discussion by danconde** [azadbolour/eagerwords-community#11] [Q&A] [answered]
The entry page has a static diagram that shows a number, exclamation mark and an asterisk (4!*).  I cannot find documentation on what some elements are, such as the exclamation mark.

**Comment by azadbolour** [comment, marked as answer]
Thanks for pointing that out. The static image is outdated and should be
replaced.

A previous version of the UI provided information about how vulnerable a
board point was to a takeover by using the symbols ! and *. However, users
did not easily remember the meaning of those indicators, and the indicators
were deemed to clutter the UI without providing the intended benefit to the
average user. So they were removed.

**Comment by azadbolour** [comment]
The image is now replaced on the beta site. Slated for release May 2023.
